cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2895,22 +2895,30 @@ alert.suppress = 0
 counttype = number of events
 quantity = 0
 relation = greater than
-cron_schedule = 2,17,32,47 * * * *
+cron_schedule = 2,32 * * * *
 description = This alert will trigger when RealTime Protection is disabled on a Sophos endpoint. \
 \
 Data Collection - Sophos Central SIEM Integration Add-on (https://splunkbase.splunk.com/app/4647/) \
 \
 A false positive may appear when an administrator might have intentionally disabled RealTime Protection for Sophos. 
-dispatch.earliest_time = -17m@m
-dispatch.latest_time = -2m@m
+dispatch.earliest_time = 0
+dispatch.latest_time = now
 display.general.type = statistics
 display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_sophos` type="Event::Endpoint::SavDisabled" \
-| stats count, latest(_time) as _time, values(src_ip) as src_ip, values(suser) as user by host, dhost \
-| eval cyences_severity = "high" \
+# Search triggers critical alerts if RealTime Protection is disabled for more than three hours, otherwise it triggers high severity alerts.
+# In order to avoid duplicate alerts having high severity, the search is running in 2 diff timeranges and merging its result.
+search = `cs_sophos` type="Event::Endpoint::Sav*" earliest=-4h@m latest=-2m@m \
```

Comment:
@hardikhdholariya I'm not sure about the logic. wouldn't this generate duplicate resuls for all events where the sophos was disabled before last 3 hours?